definitions/pipelines/rnaseq_star_fusion_with_xenosplit.cwl

```diff
@@ -0,0 +1,246 @@
+    graft_star_genome_dir:
+        type: Directory
+        doc: "Location of the STAR human reference directory"
+    graft_outfile_name_prefix:
+        type: string?
+        default: "Graft_"
+        doc: "Prefix name for the graft bam - human"
+    host_star_genome_dir:
+        type: Directory
+        doc: "Location of the STAR mouse reference directory"
+    host_outfile_name_prefix:
+        type: string?
+        default: "Host_"
+        doc: "Prefix name for the host bam - mouse"
+    star_fusion_genome_dir:
+        type: Directory
+        doc: "As the fusion would be called on only the final graftOutbam i.e. the human xenograft bam without any mouse reads we only need the star fusion directory for the human"
+    graft_gtf_file:
+        type: File
+        doc: "GTF file corresponding the to the Human STAR reference"
+    host_gtf_file:
+        type: File
+        doc: "GFT file corresponding to the Mouse STAR reference"
+    trimming_adapters:
+        type: File
+    trimming_adapter_trim_end:
+        type: string
+    trimming_adapter_min_overlap:
+        type: int
+    trimming_max_uncalled:
+        type: int
+    trimming_min_readlength:
+        type: int
+    kallisto_index:
+       type: File
+    gene_transcript_lookup_table:
+       type: File
+    strand:
+       type: string?
+    refFlat:
+        type: File
+    ribosomal_intervals:
+        type: File
+    sample_name:
+        type: string
+outputs:
+    final_bam:
+        type: File
+        outputSource: index_bam/indexed_bam
+        secondaryFiles: [.bai]
+    star_fusion_out:
+        type: File
+        outputSource: graft_star_align_fusion/chim_junc
+    star_junction_out:
+        type: File
+        outputSource: graft_star_align_fusion/splice_junction_out
+    star_fusion_log:
+        type: File
+        outputSource: graft_star_align_fusion/log_final
+    star_fusion_predict:
+        type: File
+        outputSource: star_fusion_detect/fusion_predictions
+    star_fusion_abridge:
+        type: File
+        outputSource: star_fusion_detect/fusion_abridged
+    stringtie_transcript_gtf:
+        type: File
+        outputSource: stringtie/transcript_gtf
+    stringtie_gene_expression_tsv:
+        type: File
+        outputSource: stringtie/gene_expression_tsv
+    transcript_abundance_tsv:
+        type: File
+        outputSource: kallisto/expression_transcript_table
+    transcript_abundance_h5:
+        type: File
+        outputSource: kallisto/expression_transcript_h5
+    gene_abundance:
+        type: File
+        outputSource: transcript_to_gene/gene_abundance
+    metrics:
+        type: File
+        outputSource: generate_qc_metrics/metrics
+    chart:
+        type: File
+        outputSource: generate_qc_metrics/chart
+    fusion_evidence:
+        type: File
+        outputSource: kallisto/fusion_evidence
+    xenosplit_goodnessofmapping:
+        type: File
+        outputSource: xenosplit/goodnessOfMapping
```

Comment:
Instead of "goodnessOfMapping", maybe we could call this "xenosplit_statistics"?